src/components/MenuItem.js

```diff
@@ -125,90 +125,110 @@ const MenuItem = (props) => {
-                                {props.iconType === CONST.ICON_TYPE_WORKSPACE && (
-                                    <Avatar
-                                        imageStyles={[styles.alignSelfCenter]}
-                                        size={CONST.AVATAR_SIZE.DEFAULT}
-                                        source={props.icon}
-                                        fallbackIcon={props.fallbackIcon}
-                                        name={props.title}
-                                        type={CONST.ICON_TYPE_WORKSPACE}
-                                    />
-                                )}
-                                {props.iconType === CONST.ICON_TYPE_AVATAR && (
-                                    <Avatar
-                                        imageStyles={[styles.avatarNormal, styles.alignSelfCenter]}
-                                        source={props.icon}
-                                        fallbackIcon={props.fallbackIcon}
-                                    />
-                                )}
+                    <View style={[styles.flexColumn, styles.flex1]}>
+                        {Boolean(props.label) && (
+                            <View style={props.icon ? styles.mb2 : null}>
+                                <Text style={StyleUtils.combineStyles(styles.sidebarLinkText, styles.optionAlternateText, styles.textLabelSupporting, styles.pre)}>{props.label}</Text>
                             </View>
                         )}
-                        <View style={[styles.justifyContentCenter, styles.menuItemTextContainer, styles.flex1]}>
-                            {Boolean(props.description) && props.shouldShowDescriptionOnTop && (
-                                <Text
-                                    style={descriptionTextStyle}
-                                    numberOfLines={2}
-                                >
-                                    {props.description}
-                                </Text>
+                        <View style={[styles.flexRow, styles.pointerEventsAuto, props.disabled && styles.cursorDisabled]}>
+                            {Boolean(props.icon) && _.isArray(props.icon) && (
+                                <MultipleAvatars
+                                    isHovered={hovered}
+                                    isPressed={pressed}
+                                    icons={props.icon}
+                                    size={CONST.AVATAR_SIZE.DEFAULT}
+                                    secondAvatarStyle={[
+                                        StyleUtils.getBackgroundAndBorderStyle(themeColors.sidebar),
+                                        pressed ? StyleUtils.getBackgroundAndBorderStyle(themeColors.buttonPressedBG) : undefined,
+                                        hovered && !pressed ? StyleUtils.getBackgroundAndBorderStyle(themeColors.border) : undefined,
+                                    ]}
```

Comment:
[This regression](https://github.com/Expensify/App/issues/24008) was probably caused by the PR. 

Issue: Task - Second avatar of group avatar is highlighted when long pressing on Share somewhere